web/modules/custom/dcamp/dcamp.install

```diff
@@ -39,8 +44,77 @@ function dcamp_update_8101(&$sandbox) {
  * Adjusts the URL of the sponsor prospect.
  */
 function dcamp_update_8102(&$sandbox) {
-  $config = \Drupal::configFactory()->getEditable('block.block.drupalcampblockforlandingaboutsponsoring');
+  $config = \Drupal::configFactory()
+    ->getEditable('block.block.drupalcampblockforlandingaboutsponsoring');
   $settings = $config->get('settings');
   $settings['download_url'] = 'https://drive.google.com/open?id=0B5Zv1LP5pgikSmo1dGFaRHdQWmM';
   $config->set('settings', $settings)->save();
+}
+
+/**
+ * Add dummy sponsors for easy theming and default company logo.
+ */
+function dcamp_update_8103() {
+  _dcamp_add_dummy_sponsors_and_default_logo();
+}
+
+/**
+ * Function to add some dummy content for easy theming.
+ */
+function _dcamp_add_dummy_sponsors_and_default_logo(){
+  $data = file_get_contents(drupal_get_path('module', 'dcamp') . '/images/default_company_logo.png');
```

Comment:
The images that devel generate creates are an eyesore + more overhead. I prefer not to use it.

I like the idea of putting this a demo module.

If someone can pick that up.

For the default image we do need the code in hook_install, maybe we don't need it for the sponsors, but for the speakers it would be good to have a default image.